test/test_utils.py

```diff
@@ -53,8 +53,10 @@ def is_first_device(current_device, devices):
     current_device: instance of `torch.device`.
     devices: list of device names (strings), e.g. output of
         torch_xla_py.xla_model.get_xla_supported_devices().
+    multiprocess: If true, device is ignored and only ordinal is considered.
```

Comment:
Why?

Reply:
I wanted to be explicit. I thought this would be more clear than `is_first_device(device, [str(device)])`

Reply:
The `device` arg seems irrelevant on multiprocess. It is always "xla:0"

Reply:
The semantics are the same.
You pass in the local device, and the list of all local devices.

Reply:
That's true. Switched back to original version